### PR TITLE
fix: concurrently --kill-others flag

### PR DESCRIPTION
**Pull request by bumbleblym**
**What**: fixes concurrently --kill-others flag typo

### CODE REVIEW

**Comment by kentcdodds** [comment]
Hey! Thanks for this! Don't worry about the broken build. That's unrelated stuff. Would you like to [add yourself](../blob/master/CONTRIBUTING.md#add-yourself-as-a-contributor) to the [contributors table](../#contributors)?

**Comment by bumbleblym** [comment]
updated the outdated addContributor command, and added myself as a contributor :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ This project follows the [all contributors][all-contributors] specification. To
 contributors on the README.md, please use the automated script as part of your PR:
 
 ```console
-npm start addContributor <YOUR_GITHUB_USERNAME>
+npm start "addContributor <YOUR_GITHUB_USERNAME>"
```
Comment (kentcdodds) [inline CONTRIBUTING.md]: Ah! Thank you!

**Comment by kentcdodds** [comment]
Actually, could you rebase your changes? Sorry! Things have been kinda weird because of that bug

**Comment by bumbleblym** [comment]
done; removed the `--kill-others` flag typo fix commit

**Pull request merged**